index.html

```diff
@@ -13,7 +13,7 @@ <h1 id="title">Dr. Norman Borlaug</h1>
         <img
           id="image"
           src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
-          alt=""
+          alt="People working"
         />
         <p id="img-caption">
```

Comment:
Great Job! Adding an alt text is a good html practice.  But please it would be better if you try to use class instead id because a class can be used multiple times on different elements throughout a website.